lib/contentstack/client.rb

```diff
@@ -80,13 +81,43 @@ def sync(params)
 
     private
     def get_default_region_hosts(region='us')
+      host = "https://cdn.contentstack.io" #set default host if region is nil
```

Comment:
Can you separate https protocol from the host string. Make a const variable const string PROTOCOL = "https://" and use it wherever it is required